### ISSUE TITLE
Cannot index a field with name: id when using attributes.except

### ISSUE THREAD

**Issue by seanfcarroll** [ankane/searchkick#1110]
Having a strange error, when using **attributes.except**.

```
irb(main):001:0> Sentence.reindex
  Sentence Load (13.5ms)  SELECT  "sentences".* FROM "sentences" ORDER BY "sentences"."id" ASC LIMIT $1  [["LIMIT", 1000]]
  Sentence Import (0.8ms)  {"count":1000,"exception":["Searchkick::Error","Cannot index a field with name: id"],"exception_object":"Cannot index a field with name: id"}
```

My search concern is this:

```
module SearchableSentence
  extend ActiveSupport::Concern

  included do
    searchkick callbacks: :async, word_start: [:sentence]

    def search_data
      attributes.except(:audio_file)
    end
  end

end
```

If my search concern looks like this, no problem, it indexes:

```
module SearchableSentence
  extend ActiveSupport::Concern

  included do
    searchkick callbacks: :async, word_start: [:sentence]

    #def search_data
    #  attributes.except(:audio_file)
    #end
  end

end
```

It also indexes fine if I name the columns to be searched. This used to work ... not sure what changed, I did upgrade to Rails 5.1.

**Comment by jeremiahlukus** [comment]
`gem 'searchkick', '~> 2.5'` 

Some new pushes broke what I had as well. This fixed it

**Comment by seanfcarroll** [comment]
Thanks a lot @jeremiahlukusm, that has solved the issue. I'll keep it locked at 2.5 for now. Funny thing was, I couldn't find this reported elsewhere ... perhaps some combination of Rails versions or other gems. Cheers.

**Comment by ankane** [comment]
Make sure you're on the latest version. This was fixed in 3.0.2.

**Comment by seanfcarroll** [comment]
I can confirm that it is working in 3.0.2. Thank you!
